src/VisualStudio/Core/Next/Remote/JsonRpcClient.cs

```diff
@@ -55,7 +53,10 @@ internal class JsonRpcClient : IDisposable
                 // cancellation exception here. if any exception is thrown unrelated to cancellation, then we will rethrow
                 // the exception
                 _cancellationToken.ThrowIfCancellationRequested();
-                throw;
+
+                // this is to make us not crash. we should remove this once we figure out
+                // what is causing this
```

Comment:
It is my understanding that this code, among other things, serves to prevent **devenv.exe** from crashing in the event of an unhandled exception in the OOP process. If that is the case, then this seems more like a desirable long-term solution than something temporary.